examples/step-49/doc/intro.dox

```diff
@@ -99,13 +99,337 @@ many cases it is best to combine both approaches. The file can be easily
+Circle(50) = {104, 31, 33};
+Circle(51) = {33, 31, 103};
+@endcode
+
+Below we use the <code>Circle</code> entity, which requires the origin, radius,
+and starting and ending angle respectively.
+
+@code
+rad1 = .5;
+Point(19) = {0.42, 2.0, 0, 1.0}; // origin
+Point(22) = {0.42, 2.0 + rad1, 0, 1.0}; // top point
+Point(23) = {0.42, 2.0 - rad1, 0, 1.0}; // bottom point
+Circle(12) = {0.42, 2.0, 0, rad1, 0, 2*Pi};
+@endcode
+
+<h5>Plane surfaces</h5>
+
+This section describes the "Plane Surfaces", i.e., the 2D surfaces for meshing.
+This can be found in <code>Elementary entities</code> as <code>Plane Surface</code>.
+The purpose of this entity is to specify the domain in 2D.  Because we have our
+mesh split into 4 sections, we must have 4 plane surfaces.
+
+To make a plane surface, simply click <code>Plane surface</code> and click on all
+the relevant borders for the mesh.  This includes holes or other objects contained
+in the mesh.  Again, since we have 4 different meshes, we will do this 4 times.
+
+The code below is for the top right mesh.  Notice we have 2 instances of
+<code>Curve Loop</code>.  The first one is the boundary for the outer domain
+and the second one is for the hole.  This can easily be traced back by following
+along with the numbers in the brackets.
+
+@code
+Curve Loop(1) = {40, -43, 14, 57, 59, 60, 58};
+Curve Loop(2) = {12};
+Plane Surface(1) = {1, 2};
+@endcode
+
+Surface of the bottom right partition
+
+@code
+Curve Loop(3) = {39, 52, 53, 54, 55, 56, -14, -42};
+Plane Surface(2) = {3};
+@endcode
+
+Surface of bottem left mesh
+
+@code
+Curve Loop(4) = {42, 7, 8, 9, 38};
+Curve Loop(5) = {49, 50, 51, 48};
+Plane Surface(3) = {4, 5};
+@endcode
+
+Surface of top left mesh
+
+@code
+Curve Loop(6) = {6, 43, 41, 4, 5};
+Plane Surface(4) = {6};
+@endcode
+
+<h5>Physical IDs</h5>
+
+This section is for assigning physical IDs for boundaries.  Here you press
+<code>Geometry -> Physical groups -> Add -> Curve</code>.  You may do this
+point by point but we will only be needing to use curves.  This assigns
+boundary IDs to each vertex on the boundary that we choose.  We will see
+in <code>step-49.cc</code> that this is useful for mesh refinements using
+deal.ii.
+
+We assign boundary ID 0 to outer boundary.  This is usually here by default
+but we include it anyway.
+
+@code
+Physical Curve(0) = {38, 39, 52, 53, 54, 55, 56, 57, 59, 60, 58, 40, 41, 4, 5, 6, 7, 8, 9};
+@endcode
+
+Assign boundary ID of 1 top right circle.
+
+@code
+Physical Curve(1) = {12};
+@endcode
+
+Assign boundary IDs of 2 & 3 to the top & bottom half of the bottom left
+hole respectively.  We can see here that we needed to pick two boundaries
+to make both of the physical curves.  Recall this circle is comprised of
+four circle arcs, so Physical Curve(2) is the upper half and Physical
+Curve(3) is the lower half of the lower right circle.
+
+@code
+Physical Curve(2) = {48, 49};
+Physical Curve(3) = {50, 51};
+@endcode
+
+Assigns Physical Surface to the curves
```

Comment:
as mention in the .geo file, I think this assigns a material id to the surface and has nothing to do with curves (the ids are the 4 surfaces)